applications/mne_analyze/plugins/rawdataviewer/fiffrawviewdelegate.cpp

```diff
@@ -236,7 +238,7 @@ void FiffRawViewDelegate::createPlotPath(const QStyleOptionViewItem &option,
     }
 
     qint32 kind = t_pModel->getKind(index.row());
-    double dMaxValue = 1.0e-10;
+    double dMaxValue = DISPLIB::ScalingView::getScalingValue(t_pModel->getScaling(), kind, t_pModel->getUnit(index.row()));
 
     switch(kind) {
```

Comment:
we do not need the switch case anymore right? If yes, please remove. Same geos for all other places where you now use `getScalingValue`.